src/DssSpell.t.sol

```diff
@@ -499,6 +493,20 @@ contract DssSpellTest is DssSpellTestBase {
         }
     }
 
+    function testPaymentsSum() public {
+        uint256 sumPayments = 2889;
```

Comment:
I would actually suggest to make it the same test as `testPayments`, otherwise it's easy to forget to enable this one. Moreover, `testPayments` already tests `totAmount`. You just need to add `expectedTotAmount` variable on top of the function to check if its the same as `totAmount`

Reply:
`totAmount` is computed from the test data, i would prefer to assert actual value that is the result of on-chain actions. See the adjustment